packages/dev/core/src/FlowGraph/flowGraphInteger.ts

```diff
@@ -0,0 +1,90 @@
+import { RegisterClass } from "../Misc/typeStore";
+
+/**
+ * @experimental
+ * Class that represents an integer value.
+ */
+export class FlowGraphInteger {
+    /**
+     * The value of the integer. Its type
+     * is a javascript number.
+     */
+    value: number;
+
+    constructor(value: number) {
+        this.value = this.convertToInt(value);
+    }
+
+    /**
+     * Converts a float to an integer.
+     * @param n the float to convert
+     * @returns
+     */
+    public convertToInt(n: number): number {

```

Comment:
maybe `toInt` instead of convertToInt? Might be me only, but convertToInt feels like an in-place operation that converts the value to an int and stores it as int.

Reply:
Shouldn't that be a static member?